src/Pokemon.java

```diff
@@ -0,0 +1,21 @@
+import java.util.List;
+
+public abstract class Pokemon {
+    protected int level;  // protected so it can be accessed by subclasses
```

Comment:
Je maakt van deze variabele effectief geen gebruik, omdat je in je subclasses een eigen "level" variabele definieert.